docker/jenkins/run_integration_tests.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash -xe
 GIT_COMMIT=${GIT_COMMIT:-$(git rev-parse HEAD)}
-cp docker/dockerfiles/bedrock_functional_tests Dockerfile
-docker build -t bedrock_functional_tests:${GIT_COMMIT} --pull=true .
+cp docker/dockerfiles/bedrock_integration_tests Dockerfile
+docker build -t bedrock_integration_tests:${GIT_COMMIT} --pull=true .
+
+if [ -z "${BASE_URL}" ]; then
+  # start bedrock
+  docker run -d \
+    --name bedrock-code-${BUILD_NUMBER} \
+    -e ALLOWED_HOSTS="*" \
+    -e SECRET_KEY=foo \
+    -e DEBUG=False \
+    -e DATABASE_URL=sqlite:////tmp/temp.db \
```

Comment:
Just a note: if you don't need a DB that will stick around you can just set `DATABASE_URL=sqlite://` which will cause it to use a purely in-memory database and never create a file.